### DISCUSSION TITLE
Reports per organization

### DISCUSSION THREAD

**Discussion by fkaag71** [kimai/kimai#4628] [General]
### Describe the problem

I wanted to use Kimai for a collaborative project with many partners.
Each partner manages his own team, that reports for activity on common workpackages and activities.
But at the coordinator level, I should have reports only for the global workload and expenses for each partner, without visibility on the individuals data or the unitary expenses.

### Describe the solution you'd like

- For each user, allow a "primary team", picked within the list of teams, that corresponds to his organization.
- Hide users with a primary team from all users that do not belong to this team.
- Create aggregated reports and exports per primary team, both for workload and expenses. If a user is not assigned to a primary team, he will form a team on his own.

### Describe alternatives you've considered

None so far, but if you have a workaround I will be glad to read it.

**Comment by kevinpapst** [comment]
Converted to a discussion:
- I have no clue what you want to achieve and whether it is possible already or not (I'd say it should work, but I am not sure if I understand it correctly)
- if it doesn't work: the team or permission structure won't be changed

If you can't get it up and working, consider to host multiple Kimai instances.

**Reply by fkaag71** [reply]
For a EU funded project, many partners cooperate, and an assigned coordinator has to report to the EU with workload done by each partner on each work package, and incurred costs per partner, both for human resources and other expenses. Yet the coordinator is not entitled to view the personal details or activity of the employees of other partners.
If you have a moment we can discuss it online now.

**Reply by fkaag71** [reply]
And if you are not available now, but interested to understand the need, pick a timeslot [here](https://cardynal.fr/rdv)

**Reply by kevinpapst** [reply]
Good one 😁  
You can pick a slot here: https://www.kimai.org/en/support.html
